src/main/java/transforms/GeocoderTransform.java

```diff
@@ -68,7 +68,19 @@ public String getAddress( String input, String addressObject ) {
         if ( outercodes.contains( addressObject ) ) {
             return (String) address.get( addressObject );
         } else if ( addressObject.equals("geographypoint") ) {
-            return address.get( "lat" ) + "," + address.get("lon");
+            String latitude = address.get( "lat" ).toString();
```

Comment:
It's much safer to try reading this value as a double and then write it out as a String than expect that the input values will be formatted correctly. It would also be safer to handle the case where address doesn't contain the lat or lon key (otherwise .toString() will throw an NPE). I'd also recommend moving these string constants into static constants at the top of this file.

 i.e.
```
private static final String LATITUDE = "lat";
private static final String LONGITUDE = "lon";
private static final String GEO_POINT_KEY = "geographypoint";

...

private static Double getDoubleFromObj( Object o ) {
  if ( o == null ) {
    return null;
  }

  return Parsers.parseDouble( o.toString() );
}

} else if ( addressObject.equals( GEO_POINT_KEY ) ) {
  
  Double latitude = getDoubleFromObj( address.get( LATITUDE ) );
  Double longitude = getDoubleFromObj( address.get( LONGITUDE ) );

  if ( latitude == null || longitude == null) {
    return null;
  }

  return latitude.toString() + "," + longitude.toString();
}
```